changelogutils/validator_test.go

```diff
@@ -721,71 +744,6 @@ var _ = Describe("changelog validator utils", func() {
 			})
 		})
 
-		Context("invalid settings", func() {
```

Comment:
this section proved tedious to edit, as it relied on a number of hardcoded mocks expected to be called N times each.  I found it easier to just _not_ adjust it